mentoring-app/src/app/header/header.component.html

```diff
@@ -3,7 +3,7 @@
         <img src="assets/images/courses-logo.png" alt="logo">
     </div>
     <div class="header__buttons">
-            <button mat-button color="primary">Log in</button>
-            <button mat-button color="primary">Log off</button>
+            <button *ngIf="!authService.isAuthenticated()" mat-button>Log in</button>
```

Comment:
*ngIf else construction can be used here to avoid double evaluation.

Reply:
done